feat(content): Continuation to Quicksilver package mixup (#8366)
endless-sky/endless-sky: data/human/human missions.txt

				die
			label skip
			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`

mission "Quicksilver Mixup [2]"
	name "Misposted mail delivery"
	description "Carry misposted mail from Silver to Quicksilver by <date> and then come back with the return shipment"
	minor
	deadline 10 1
	source "Silver"
	destination "Quicksilver"
	cargo "Misposted mail" 38
	to offer

> 		has "Quicksilver Mixup [1]: done"

Reviewer: ```suggestion
		has "Quicksilver Mixup [1]: done"
		has "quicksilver mixup 2 available"
```

	on offer
		conversation
			`As you walk through Silver's spaceport, somebody wearing office attire power walks up to you after ending a hurried conversation with another pilot.`

> 			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to Quicksilver and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day>? We'll pay you what he'd be paid, 100.000 credits."`

Reviewer 2: ```suggestion
			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to Quicksilver and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day>? We'll pay you what he'd be paid, 100,000 credits."`
```

Author: Oh yeah, in english ya'll use a comma there

				die
			label skip
			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`

mission "Quicksilver Mixup [2]"
	name "Misposted mail delivery"
	description "Carry misposted mail from Silver to Quicksilver by <date> and then come back with the return shipment"
	minor
	deadline 10 1
	source "Silver"
	destination "Quicksilver"

> 	cargo "Misposted mail" 38

Reviewer: ```suggestion
	cargo "misposted mail" 38
```
I'm not sure if this is necessary, but generally they're in lower case.

Author: Necessary or not, if they're generally in lower case, i see no reason not to!

	to offer
		has "Quicksilver Mixup [1]: done"
		has "quicksilver mixup 2 available"
	on offer
		conversation
			`As you walk through Silver's spaceport, somebody wearing office attire power walks up to you after ending a hurried conversation with another pilot.`

> 			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to Quicksilver and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day>? We'll pay you what he'd be paid, 100.000 credits."`

Author: Is there any advantage or disadvantage to having it written one way or the other? Seems like it would allow you to edit the payment by changing one as opposed to two lines, without cons. I'd say this is a fine change, right?

Either way, i will wait for a third person to check before committing any of these, as you asked.

> 			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to Quicksilver and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day>? We'll pay you what he'd be paid, 100.000 credits."`

Reviewer: There's no disadvantage as far as I know.

> 			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to Quicksilver and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day>? We'll pay you what he'd be paid, 100.000 credits."`

Author: Thank you, i'll commit it then.

			choice
				`	"Oh! So you guys did set up that regular shipment, after all!"`
					goto context

				`	"Sure, how much mail are we talking about?"`
					goto shipment

				`	"Sorry, but i'm not heading in that direction"`
					goto refuse

> 			label context

Author: I personally prefer the labels for readability, but that's minor - if clearing up lines has a benefit to the code, I'm fine with it!

> 			`	The mail employee appears a tad confused for a moment, but then you mention you once delivered a misposted package like that, and the clerk had mentioned somebody should set up a regular delivery due to how common of an issue it was. The post employee smiles and responds: "Oh! That was me, actually, back when I worked as a clerk for the post office. I was sick of always having that same conversation with so many pilots a day, so I went out and found a captain to set up a regular delivery with and made the proposal to my boss, who was just as sick of my and the Quicksilver's postmaster office endless reports. Got a bonus and promotion for it - I'm now responsible for the delivery's management, instead! Anyways, are you interested in covering for them?"`

Reviewer: ```suggestion
			`	The mail employee pulls a wierd face at you for a moment.
			choice
				`	"I once delivered a misposted package like that, and the clerk mentioned that it happened so often somebody should set up a regular delivery."`
				`	"Never mind. What's the size of the shipment?"`
					goto shipment
			`	The post employee smiles and responds, "Oh! That was me, actually, back when I worked as a clerk for the post office. I was sick of always having that same conversation with so many pilots a day, so I went out and found a captain to set up a regular delivery with and made the proposal to my boss, who was just as sick of my and the Quicksilver postmaster's endless reports. Got a bonus and promotion for it; I'm now responsible for the delivery's management, instead! Anyways, are you interested in covering for them?"`
```
The choice might be far too long, but I'll suggest it anyway. Don't commit it unless there's someone else that can comment about it.

Reviewer: I like the second one better, because I want to show as much of the situation, not just tell the player that the employee's confused.

Author: Nice, let's do that, then!

			choice
				`	"Yeah, what's the size of the shipment?"`

> 					goto shipment

Reviewer: ```suggestion
```

Author: Same as before, if it helps, i'll definitely remove the line, but if it doesn't make a difference to performance, file sizes and so on, i prefer it for readability.

				`	"Sorry, but I'm not able to work that into my plans."`
					goto refuse

			label shipment

> 			`	The man checks his notes and says "This week, it totalled... 38 tons of letters and miscellaneous packages here, and then 29 tons to be brought back from Quicksilver."`

Author: Is "checks his notes" cliché in English? I didn't know. 

Personally, I find "flips through" to imply disorganisation, like somebody hurriedly throwing around the pages of a notebook, and with this guy being a seemingly organised clerk-turned-manager, it doesn't fit imo. I am fine with changing it to something else neutral, that imply his notes are easily accessible, just requiring him to find the specific line with the info again. 

What do you think of "The man takes a look at his tablet and says/says,"? It could be added to the start of the mission, as well ("somebody wearing office attire **and carrying a tablet** power walks up...")

About the comma, I'm fine with it either way.

> 			`	The man checks his notes and says "This week, it totalled... 38 tons of letters and miscellaneous packages here, and then 29 tons to be brought back from Quicksilver."`

Reviewer: Checks notes is a weird joke in English I've never got my head around. The tablet idea sounds good though.

The comma is necessary anyway : )

```suggestion
			`	The man takes a look at his tablet and says, "This week, it totalled... 38 tons of letters and miscellaneous packages here, and then 29 tons to be brought back from Quicksilver."`
```

> 			`	The man checks his notes and says "This week, it totalled... 38 tons of letters and miscellaneous packages here, and then 29 tons to be brought back from Quicksilver."`

Author: Neat! I'll commit it and add the tablet line to the start of the mission, too!

			`	"38 tons of misposted packages? Just this month?!" you ask.`

> 			`	The man nods "Told you it happened often. And few of these packages even weigh over a kilo! Big shipments hardly get misposted, it's mostly private citizens who make such a mistake. Mostly. You'd be amazed!"`

Author: This one i quite like and is minor enough i'll just commit it.

> 			`	You two shake hands and you tell him which landing pad <ship> is parked. Time to head to Quicksilver.`

Author: I quite like when the game mentions your ship by name - perhaps "arrange for the misposted packages to be loaded onto the <ship>" would be good?

> 			`	You two shake hands and you tell him which landing pad <ship> is parked. Time to head to Quicksilver.`

Reviewer: ```suggestion
			`	You two shake hands and arrange for the misposted packages to be loaded onto the <ship>. Time to head to Quicksilver.`
```

				accept

			label refuse

> 			`	The mail employee thanks you for your time, and continues his search for a willing captain. At least pilots that get hired to deliver such wrongly-posted packages, like you once did, won't have to finish those deliveries themselves in the future!`
> 				accept

Reviewer: ```suggestion
			`	The mail employee thanks you for your time, and continues his search for a willing captain. At least pilots that get hired to deliver such wrongly-posted packages, like you once did, won't have to finish those deliveries themselves in the future.`
				decline
```

	on visit
		dialog `You have reached <planet>, but your escort, loaded with the shipment, has not arrived. Better depart and wait for your escorts to arrive in this star system.`

mission "Quicksilver Mixup [3]"
	name "Misposted mail delivery"
	deadline 10 1
	landing
	description "Return the misposted mail back to <destination>."
	source "Quicksilver"
	destination "Silver"

> 	cargo "Misposted mail" 29

Reviewer: ```suggestion
	cargo "misposted mail" 29
```

				die
			label skip
			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`

mission "Quicksilver Mixup [2]"
	name "Misposted mail delivery"

> 	description "Carry misposted mail from Silver to Quicksilver by <date> and then come back with the return shipment"

Reviewer: ```suggestion
	description "Deliver <tons> of misposted mail from <origin> to <planet> by <date> and then come back with the return shipment. You will be paid <payment>."
```

	minor
	deadline 10 1
	source "Silver"
	destination "Quicksilver"

> 	cargo "misposted mail" 38

Reviewer: ```suggestion
	cargo "misposted mail" 38
	"apparent payment" 100000
```

	to offer
		has "Quicksilver Mixup [1]: done"
		has "quicksilver mixup 2 available"
	on offer
		conversation
			`As you walk through Silver's spaceport, somebody wearing office attire and carrying a tablet power walks up to you after ending a hurried conversation with another pilot.`

> 			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to Quicksilver and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day>? We'll pay you what he'd be paid, <payment>."`

Reviewer 2: ```suggestion
			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to Quicksilver and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day>? We'll pay you what he'd be paid, 100,000 credits."`
```
@bene-dictator There's no payment on this mission! That's why it needs to be hard-coded, it's only on the follow-up mission.

Reviewer: Use "apparent payment" then.

			choice
				`	"Oh, so you guys did set up that regular shipment after all!"`
					goto context

				`	"Sure, how much mail are we talking about?"`
					goto shipment

				`	"Sorry, but I'm not heading in that direction."`
					goto refuse

			label context

> 			`	The mail employee appears confused for a moment, leaning his head to the side slightly.

Reviewer 2: ```suggestion
			`	The mail employee appears confused for a moment, leaning his head to the side slightly.`
```
Missing quote. If you're putting together more missions in the future, I'd definitely recommend Visual Studio Code with the Endless Sky extension. That makes it pretty apparent when quotes aren't where they need to be.
![image](https://user-images.githubusercontent.com/4471575/218886304-543be4d2-7cea-45a8-bdb5-c4b5aab616b9.png)
![image](https://user-images.githubusercontent.com/4471575/218886332-5724cba9-a4c9-435f-a444-f5206d5a51b3.png)

Reviewer: Hahaha that was me, I forgot it while doing the suggestion : )

					goto refuse

			label shipment
			`	The man takes a look at his tablet and says, "This week, it totalled... 38 tons of letters and miscellaneous packages here, and then 29 tons to be brought back from Quicksilver."`
			`	"38 tons of misposted packages? Just this month?!" you ask.`
			`	He nods. "Told you it happened often. And few of these packages even weigh over a kilo! Big shipments hardly get misposted, it's mostly private citizens who make such a mistake. Mostly. You'd be amazed!"`
			`	You two shake hands and arrange for the misposted packages to be loaded onto the <ship>. Time to head to Quicksilver.`
				accept

			label refuse
			`	The mail employee thanks you for your time, and continues his search for a willing captain. At least pilots that get hired to deliver such wrongly-posted packages, like you once did, won't have to finish those deliveries themselves in the future.`
				decline
	on visit
		dialog `You have reached <planet>, but your escort, loaded with the shipment, has not arrived. Better depart and wait for your escorts to arrive in this star system.`

mission "Quicksilver Mixup [3]"
	name "Misposted mail delivery"
	deadline 10 1
	landing

> 	description "Return the misposted mail back to <destination>."

Reviewer: ```suggestion
	description "Return to <planet> with <tons> of misposted mail. You will be paid <payment>."
```

	source "Quicksilver"
	destination "Silver"
	cargo "misposted mail" 29
	to offer

> 		has "Quicksilver Mixup [2]"

Reviewer 2: ```suggestion
		has "Quicksilver Mixup [2]: done"
```

	on offer

> 		dialog `You contact the spaceport authorities and they direct you to the respective landing pad, where the post office staff is already waiting. They transfer the cargo and you prepare to head back to Silver. You, again, have 10 days to make the delivery.`

Reviewer: ```suggestion
		dialog `You contact the spaceport authorities and they direct you to the respective landing pad, where the post office staff is already waiting. They transfer the cargo and you prepare to head back to Silver. Again, you have 10 days to make the delivery.`
```
This is easier to read.

Author: I am guilty of loving inversions, heheheh
But yeah, i'll commit the simpler version. It is easier to read.

			

event "quicksilver mixup 2 available"

mission "Quicksilver Mixup 2"

> 	name "Misposted mail delivery"

Reviewer 3: ```suggestion
	name "Misposted Mail Delivery"
```

	description "Deliver <tons> of misposted mail from <origin> to <planet> by <date> and then come back with the return shipment. You will be paid <payment> after you return from <planet>."
	minor
	deadline 10 1
	source "Silver"
	destination "Quicksilver"
	cargo "misposted mail" 38
	"apparent payment" 100000
	to offer
		has "Quicksilver Mixup 1: done"
		has "quicksilver mixup 2 available"
	on offer
		conversation

> 			`As you walk through Silver's spaceport, somebody wearing office attire and carrying a tablet power walks up to you after ending a hurried conversation with another pilot.`

Reviewer 3: ```suggestion
			`As you walk through <origin>'s spaceport, somebody wearing office attire and carrying a tablet power walks up to you after ending a hurried conversation with another pilot.`
```

> 			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to Quicksilver and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day>? We'll pay you what he'd be paid, <payment>."`

Reviewer 3: ```suggestion
			`	"Captain! I work for the postmaster's office. The regular pilot who carries misposted mail to <planet> and back couldn't make it this month, some mechanical issues with his ship, I'm told. Would you be willing to carry this month's load there by <day> and then return with the mail from <planet>? We'll pay you what he'd be paid: <payment>."`
```

			choice
				`	"Oh, so you guys did set up that regular shipment after all!"`
					goto context

				`	"Sure, how much mail are we talking about?"`
					goto shipment

				`	"Sorry, but I'm not heading in that direction."`
					goto refuse

			label context
			`	The mail employee appears confused for a moment, leaning his head to the side slightly.`
			choice
				`	"I once delivered a misposted package like that, and the clerk mentioned that it happened so often somebody should set up a regular delivery."`
				`	"Never mind. What's the size of the shipment?"`
					goto shipment

> 			`	The post employee smiles and responds, "Oh! That was me, actually, back when I worked as a clerk for the post office. I was sick of always having that same conversation with so many pilots a day, so I went out and found a captain to set up a regular delivery with and made the proposal to my boss, who was just as sick of my and the Quicksilver postmaster's endless reports. Got a bonus and promotion for it; I'm now responsible for the delivery's management, instead! Anyways, are you interested in covering for them?"`

Reviewer 3: ```suggestion
			`	The post employee smiles and responds, "Oh! That was me, actually, back when I worked as a clerk for the post office. I was sick of always having that same conversation with so many pilots a day, so I went out and found a captain to set up a regular delivery with and made the proposal to my boss, who was just as sick of my and the <planet> postmaster's endless reports. Got a bonus and promotion for it; I'm now responsible for the delivery's management, instead! Anyways, are you interested in covering for them?"`
```

			choice
				`	"Yeah, what's the size of the shipment?"`
					goto shipment
				`	"Sorry, but I'm not able to work that into my plans."`
					goto refuse

			label shipment

> 			`	The man takes a look at his tablet and says, "This week, it totaled... 38 tons of letters and miscellaneous packages here, and then 29 tons to be brought back from Quicksilver."`

Reviewer 3: ```suggestion
			`	The man takes a look at his tablet and says, "This week, it totaled... <tons> of letters and miscellaneous packages here, and then 29 tons to be brought back from Quicksilver."`
```

> 			`	"38 tons of misposted packages? Just this month?!" you ask.`

Reviewer 3: ```suggestion
			`	"<tons> of misposted packages? Just this month?!" you ask.`
```

> 			`	He nods. "Told you it happened often. And few of these packages even weigh over a kilo! Big shipments hardly get misposted, it's mostly private citizens who make such a mistake. Mostly. You'd be amazed!"`

Reviewer 3: ```suggestion
			`	He nods. "Told you it happened often. And only a few of these packages even weigh over a kilo! Big shipments hardly get misposted; it's mostly private citizens who make these kinds of mistakes. Mostly. You'd be amazed!"`
```

> 			`	You two shake hands and arrange for the misposted packages to be loaded onto the <ship>. Time to head to Quicksilver.`

Reviewer 3: ```suggestion
			`	You two shake hands and arrange for the misposted packages to be loaded onto the <ship>. Time to head to <planet>.`
```

				accept

			label refuse
			`	The mail employee thanks you for your time, and continues his search for a willing captain. At least pilots that get hired to deliver such wrongly-posted packages, like you once did, won't have to finish those deliveries themselves in the future.`
				decline
	on visit
		dialog `You have reached <planet>, but your escort, loaded with the shipment, has not arrived. Better depart and wait for your escorts to arrive in this star system.`
	on complete

> 		dialog `You contact the spaceport authorities and they direct you to the respective landing pad, where the post office staff is already waiting. They transfer the cargo and you prepare to head back to Silver. Again, you have 10 days to make the delivery.`

Reviewer 3: ```suggestion
		dialog `You contact the spaceport authorities and they direct you to the respective landing pad, where the post office staff is already waiting. They transfer the cargo and you prepare to head back to <origin>. Again, you have 10 days to make the delivery.`
```

mission "Quicksilver Mixup 3"

> 	name "Misposted mail delivery"

Reviewer 3: ```suggestion
	name "Misposted Mail Delivery"
```

	deadline 10 1
	landing

> 	description "Return to <planet> with <tons> of misposted mail. You will be paid <payment>."

Reviewer 3: ```suggestion
	description "Return to <planet> with <cargo>. You will be paid <payment>."
```

	source "Quicksilver"
	destination "Silver"
	cargo "misposted mail" 29
	to offer
		has "Quicksilver Mixup 2: done"
	on visit
		dialog `You have reached <planet>, but your escort carrying the misposted mail has not arrived. Better depart and wait for your escorts to arrive in this star system.`
	on complete
		payment 100000

> 		dialog `The same manager from before is waiting with the unloading crew and thanks you for covering for the other pilot, before handing you the credit chip for the payment. It's nice to know that captains who get hired to deliver such wrongly-posted packages, like you once did, won't have to finish those deliveries themselves in the future."`

Reviewer 3: ```suggestion
		dialog `The same manager from before is waiting with the unloading crew, and thanks you for covering for the other pilot before handing you the credit chip for payment. It's nice to know that captains who get hired to deliver such wrongly-posted packages, like you once did, won't have to finish those deliveries themselves in the future."`
```

event "quicksilver mixup 2 available"

mission "Quicksilver Mixup 2"
	name "Misposted Mail Delivery"
	description "Deliver <tons> of misposted mail from <origin> to <planet> by <date> and then come back with the return shipment. You will be paid <payment> after you return from <planet>."
	minor
	deadline 10 1
	source "Silver"
	destination "Quicksilver"
	cargo "misposted mail" 38
	"apparent payment" 100000
	to offer
		has "Quicksilver Mixup 1: done"

> 		has "quicksilver mixup 2 available"

Reviewer 3: ```suggestion
		has "event: quicksilver mixup 2 available"
```
Caught this while testing. Need that `event` bit in front to use this as a condition.

Author: Nice, thanks!
